sdk/core/azure-core-http-okhttp/src/main/java/com/azure/core/http/okhttp/OkHttpAsyncHttpClientBuilder.java

```diff
@@ -212,6 +213,20 @@ public OkHttpAsyncHttpClientBuilder configuration(Configuration configuration) {
         return this;
     }
 
+    /**
+     * Sets the followRedirect flag on the underlying OkHttp-backed {@link com.azure.core.http.HttpClient}.
+     *
+     * If this policy is set to 'true' OkHttp-backed {@link com.azure.core.http.HttpClient} will follow redirects automatically
+     * If your HTTP pipeline is configured to call redirect policy it will not be called if this value is true.
```

Comment:
```suggestion
     * If this is set to 'true' redirects will be followed automatically, and
     * if your HTTP pipeline is configured with a redirect policy it will not be called.
```

Thoughts on being a bit more direct and not adding context that is inferred from the API being called?